### ISSUE TITLE
Bug: Error: Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2)

### ISSUE THREAD

**Issue by henkvalk** [weprovide/valet-plus#590]
- [x] I've checked the issue queue and could not find anything similar to my bug.
- [x] I'm on the latest version of valet-plus (`valet --version`): `Valet+ 2.3`
- [x] I've run `valet fix` and `valet install` after updating and before submitting my issue/feature.

**What is the problem?**
Error on mysql command. 

**What was supposed to happen?**
mysql should run from command line

**What actually happened?**
it returns an error:
```
Error: Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2)
```

**How to reproduce this?**
- A clean install of macOS Monterey on a Mac Silicon (M1) 
- `brew tap henkrehorst/php`
- `brew install valet-php@7.3 --build-from-source`
- `brew install valet-php@7.4 --build-from-source`
- `brew link valet-php@7.4 --force`
- `brew install composer`
- `composer global require weprovide/valet-plus`
-  disable geoip inside `vi ~/.composer/vendor/weprovide/valet-plus/cli/Valet/Pecl.php`
- Ad pcre2 to profile
```
echo 'export LDFLAGS="-L/opt/homebrew/opt/pcre2/lib"' >> ~/.zshrc
echo 'export CPPFLAGS="-I/opt/homebrew/opt/pcre2/include"' >> ~/.zshrc
echo 'export PKG_CONFIG_PATH="/opt/homebrew/opt/pcre2/lib/pkgconfig"' >> ~/.zshrc
ln -s /opt/homebrew/include/pcre2.h /opt/homebrew/Cellar/valet-php@7.3/7.3.27_2/include/php/ext/pcre/pcre2.h
```
`valet fix && valet install --with-mariadb` or `valet fix && valet install --with-mysql-8`
**What is the solution?**
no one found

**Sources**
All sources related to the bug. If the bug uses external tools like PHP extensions it should at
least contain a link to the tool. Any other media which proves helpful can be included here.

**Comment by claudiusararu** [comment]
👍 i can confirm that @Tjitse-E 's typo fix also fixes the `Error: Can't connect to local MySQL server through socket '/tmp/mysql.sock'`  issue. At least it did for me. Intel Version, mysql 8.

**Comment by Tjitse-E** [comment]
@henkvalk  can you close this issue? it was fixed in #596 .

**Comment by henkvalk** [comment]
Resolved in #596
